config/base_all.toml

```diff
@@ -1,10 +1,14 @@
 [parameters]
 seed = 0
-region = "leeds"
+region = "greater-london"
 zone_id = "OA21CD"
-travel_times = true # Only set to true if you have travel time matrix at the level specified in boundary_geography
+travel_times = false      # Only set to true if you have travel time matrix at the level specified in boundary_geography
 boundary_geography = "OA"
 
+[matching]
+load_hh = false
```

Comment:
Could you add a comment here to describe what this does? I don't understand how it is used

Reply:
As discussed, we'll remove this from the config and replace with non-config variables to enable use for debugging and potentially split into multiple scripts instead in a new issue.

Reply:
Removed in [72c7c21](https://github.com/Urban-Analytics-Technology-Platform/acbm/pull/61/commits/72c7c218dc273d786dbdff68473e55afe2a1f067)